CHANGELOG.rst

```diff
@@ -22,4 +22,12 @@ Unreleased
 Added
 =====
 
-* First release on PyPI.
+* Initial section-derived course creation code.
+
+[0.2.0] - 2023-05-04
+************************************************
+
+Added
+=====
+
+* Admin views for creating new section-based courses.
```

Comment:
Nit: most changelogs have newer entries at the top because users are usually interested in reading the latest changes. Scrolling through all of them is a bad user experience.